spec/features/issues/issues_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'spec_helper'
+
+feature 'Issues' do
+	let!(:user) { create :user }
+
+	scenario 'creation' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Tab detected.